modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/viewmodel/AdvancedSettingsViewModel.kt

```diff
@@ -68,13 +61,12 @@ class AdvancedSettingsViewModel
     }
 
     data class State(
-        val refreshPodcastsAutomatically: Boolean,
         val backgroundSyncOnMeteredState: BackgroundSyncOnMeteredState
     ) {
 
         data class BackgroundSyncOnMeteredState(
-            @StringRes val summary: Int,
             val isChecked: Boolean = true,
```

Comment:
Can we remove this `isChecked` default value of `true`? This value should always be populated from the user's settings so I think we should force the caller to pass it.